java/Cherepnin_Oleh/src/com/zoolatech/lecture10/tasks/FileReader.java

```diff
@@ -0,0 +1,43 @@
+package com.zoolatech.lecture10.tasks;
+
+import java.io.*;
+import java.util.Date;
+
+public class FileReader {
+    private String path;
+
+    public FileReader(String path) {
+        this.path = path;
+    }
+
+    public void readFile(String name) {
+        StringBuilder builder = new StringBuilder(path);
```

Comment:
nit: StringBuilder can be a better option in some cases. Ex concatenation in a loop. If you need to concatenate 2-3 strings + will be better